doc/source/whatsnew/v1.1.4.rst

```diff
@@ -14,7 +14,7 @@ including other versions of pandas.
 
 Fixed regressions
 ~~~~~~~~~~~~~~~~~
--
+- Fixed regression in :func:`merge` on merging datetime index with empty :class:`DataFrame` (:issue:`36895`)
```

Comment:
move to 1.2; this has gone on long enough that we can't change in a point release.